docs/content/users/quickstart.md

```diff
@@ -292,6 +292,28 @@ DDEV comes ready to work with any PHP project, and has deeper support for severa
     
     Magento 2 is a huge codebase, and we recommend [using Mutagen for performance](install/performance.md#using-mutagen) on macOS and traditional Windows.
 
+=== "Moodle"
+
+    ## Moodle
+    
+    ```bash
+    ddev config --composer-root=public --create-docroot --docroot=public --webserver-type=apache-fpm
+    mkdir .ddev/php
+    echo "max_input_vars = 5000;" > .ddev/php/custom.ini
```

Comment:
```suggestion
```

Can be removed once https://github.com/drud/ddev/pull/4300 is merged.

Reply:
#4300 has been merged, so we can remove these lines.